OracleJava/README.md

```diff
@@ -1,21 +1,21 @@
 Oracle Java on Docker
 =====
-This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of currently supported versions 11 and later and for Server JRE 8.
+This repository contains sample Docker configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux with JDK images of JDK 16, 15, and 11 and for Server JRE 8.
 
 Oracle Java Server JRE provides the features from Oracle Java JDK commonly required for server-side applications (i.e. Running a Java EE application server). For more information about Server JRE, visit the [Understanding the Server JRE blog entry](https://blogs.oracle.com/java-platform-group/understanding-the-server-jre) from the Java Product Management team.
 
 ## Building the Oracle Java base image
 Download the linux x-64 compressed archive (tar.gz) [JDK or Server JRE](https://www.oracle.com/javadownload) for the version you want to create an image of and place it in the same directory as the corresponding Dockerfile.
 
-e.g. for JDK 15 download jdk-15[X]_linux-x64_bin.tar.gz into OracleJava/15, 
+e.g. for JDK 16 download jdk-16[X]_linux-x64_bin.tar.gz into OracleJava/16, 
 for ServerJRE 8 download server-jre-8uXXX-linux-x64.tar.gz into OracleJava/8
 
 Navigate to the folder containing the download and run docker build. Tag it with the correct version number.
```

Comment:
Please wrap "docker build" in backtick quotes.

Reply:
Done